### PR TITLE
fix: update vscode version to support prelease

### PR DESCRIPTION
**Pull request by kpodsiad**
I missed that prelease requires vscode `^1.63` but we don't want to force other users to update. Maybe what I propose is dirty, but it keeps status quo for release version and enables us to publish pre-release ones

### CODE REVIEW

**Review by tgodzik — CHANGES_REQUESTED** [review]
I think this is too soon to update.

**Review by tgodzik — APPROVED** [review]
Not the most fun thing to do, but it's better than updating it for all users. Thanks!

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -36,3 +42,6 @@ jobs:
           preRelease: true
           yarn: true
       # Don't publish for VSX, it doesn't support pre-release versions
+      # Push changes - increased patch version
+      - name: Push changes
+        run: git push origin HEAD:main
```
Comment (tgodzik) [inline .github/workflows/pre-release.yml:47]:
```suggestion
        run: git push origin HEAD
```

Comment (kpodsiad) [reply .github/workflows/pre-release.yml:47]: As we speak, I'll leave this as it is.

**Pull request merged**